pkg/helm/portforwarder/portforwarder.go

```diff
@@ -44,6 +44,16 @@ func New(namespace string, client kubernetes.Interface, config *rest.Config) (*k
 	return t, t.ForwardPort()
```

Comment:
Can you change `New` to call `NewWithPort(namespace, 44134, client, config)` so the code is a little more DRY?

Reply:
That makes far more sense! :100: